### ISSUE TITLE
Fix windows and linux builds

### ISSUE THREAD

**Issue by kpgalligan** [touchlab/SQLiter#114]
Linking for Windows and Linux not working in GitHub Actions.

**Comment by hbmartin** [comment]
I just tried with Ubuntu 20.04 and 22.04 and both seemed to work https://github.com/hbmartin/SQLiter/actions/workflows/build.yml

Odd since I noticed you already tried this 🤔

**Comment by hbmartin** [comment]
I tried a bunch of things to resolve this at the link above without success. Out of ideas for now.
But... came across another lib https://github.com/ctripcorp/SQLlin (that appears to be a derivative of this one!) with a very similar setup (including 1.9.20) that has no problem with the linker: 
https://github.com/hbmartin/SQLlin/actions/runs/6778699164/job/18424592683

**Comment by kpgalligan** [comment]
Yeah, I saw that recently. Apparently, I let [a PR sit for quite a while](https://github.com/ctripcorp/SQLlin/blob/main/sqllin-driver/README.md#design) (sorry :) ). Will take a look.
